README.md

```diff
@@ -108,7 +108,7 @@ git clone https://github.com/xcp-ng-rpms/xapi.git
 # ... Here add your patches ...
 
 # Build.
-/path/to/run.py -b 7.5 --buildlocal xapi/ --rm
+/path/to/run.py -b 7.6 -l xapi/ --rm
```

Comment:
I'd prefer --build-local (which is then mentioned to in the next paragraph)

Reply:
--build-local is also fine. 

Currently it is `--buildlocal` which errors as `unrecognized arguments`

Reply:
Yeah, I understood that :). Can you amend your commit and force-push it?

Reply:
Hmm.. Should we be changing all `7.5` to `7.6`?

Reply:
You can if you wish so (there will also be changes in the help of run.py itself)